src/Http/Requests/Bulk/DestroyThreads.php

```diff
@@ -33,29 +33,44 @@ public function authorizeValidated(): bool
 
     public function fulfill()
     {
-        $threads = $this->threads();
+        $threads = $this->threads()->get();
+
+        if ($threads->count() === 0) return 0;
+        
+        // Avoid using Eloquent to prevent automatic touching of updated_at
+        $query = \DB::table((new Thread)->getTable())
+            ->whereNull('deleted_at')
+            ->whereIn('id', array_unique($this->validated()['threads']));
 
-        if (config('forum.general.soft_deletes') && isset($this->validated()['permadelete']) && $this->validated()['permadelete'] && method_exists(Thread::class, 'forceDelete'))
+        if (config('forum.general.soft_deletes') && isset($this->validated()['permadelete']) && $this->validated()['permadelete'])
         {
-            $threads->forceDelete();
+            $query->delete();
         }
         else
         {
-            $threads->delete();
+            $query->update(['deleted_at' => \DB::raw('NOW()')]);
```

Comment:
Just realised this soft-deletes if `forum.general.soft_deletes` is set to false. Will fix later.